core/tests/protractor/featureGatingFlow.js

```diff
@@ -86,7 +86,9 @@ describe('Feature Gating Flow', function() {
     await users.logout();
   });
 
-  it('should show indicators after enabling dummy_feature', async() => {
+  // TODO(#12297): Re-enable once we've found an alternative session ID that can
+  // persist across disjoint user sessions.
+  xit('should show indicators after enabling dummy_feature', async() => {
```

Comment:
There's going to be a new linter task to block usages of `xit`. Is this feature no longer available? If so, can we remove the test until we add the feature back?

Reply:
Done.